.github/workflows/utils/sql-proxy.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -ex
+
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Shared Cloud SQL Proxy setup
+# Presumes the following variables in ci-setup.json:
+#  * CLOUD_SQL_CONNECTION_NAME - the project:region:instance of a Cloud SQL instance.
+#  * UNIX_SOCKET_DIR - a local directory to set the proxy to (default tmp/cloudsql)
+#
+# Note: in GitHub Actions environments, `/cloudsql` is not valid.
+# Ensure any INSTANCE_UNIX_SOCKET value is ~= $UNIX_SOCKET_DIR/$CLOUD_SQL_CONNECTION_NAME
+#
+# Usage:
+#    sql-proxy.sh [..]
+#
+# Defaults to TCP socket proxy. Set `SOCKET=unix` for Unix sockets.
+#
+# Usage in package.json:
+#
+#    "proxy": "bash $GITHUB_WORKSPACE/.github/workflows/utils/debug-sql-proxy.sh",
```

Comment:
"proxy": "bash -ex ..." (see comment below)

Reply:
Will update based on changes to above comment